frontend/src/assets/manual/en-GB/citation.md

```diff
@@ -0,0 +1,36 @@
+## Citing documents from I-analyzer
+
+If you want to cite a document that you accessed through I-analyzer, be sure to read the documentation of the corpus. This may include more specific information about the data and how to cite it. This page gives some general advice on citing documents from I-analyzer in your research.
+
+### Citing a document from the I-analyzer interface
+
+If you downloaded a document in I-analyzer, or read it online in the I-analyzer interface, your citation should make it clear that you retrieved it through here. This provides accountability: loading corpora into I-analyzer is a process that may introduce mistakes, so you should always cite the resource that you actually referenced.
+
+For example, a citation might look something like the following:
+
+> Doe, J. (1900, Jan 1). A very interesting article. *The Times.* http://ianalyzer.hum.uu.nl/document/fascinating-corpus/12345678
+
+The specifics will depend on the citation style you are using and the type of data in the corpus.
+
+Documents in I-analyzer have a unique link, which looks like the one in the example above. This is the most appropriate way to reference a specific document.
+
+### Citing collection of documents
+
+For distant reading research, you may want to cite a large collection of documents, rather than a few specific ones. In this case, it is usually most appropriate to cite the corpus as a whole. This may look something like the following:
+
+> Research Software Lab, Centre for Digital Humanities, Utrecht University (2023). *The Times* [data set]. http://ianalyzer.hum.uu.nl/search/times
```

Comment:
Again, the Times is not a resource all users will have access to.

Reply:
This is more by way of example, but a public corpus would be clearer. I will update it.